devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -72,6 +72,9 @@ metadata {
 		// Third Reality
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS008Z", deviceJoinName: "RealitySwitch Plus"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS007Z", deviceJoinName: "RealitySwitch"
+
```

Comment:
We're organizing these in alphabetical order now instead of putting new ones at the bottom. This makes it easier to find a fingerprint plus decreases the change that there will be a merge conflict on the PR.

Reply:
I moved the fingerprint